source/extensions/access_loggers/filters/cel/config.cc

```diff
@@ -0,0 +1,65 @@
+#include "source/extensions/access_loggers/filters/cel/config.h"
+
+#include "envoy/extensions/access_loggers/filters/cel/v3/cel.pb.h"
+
+#include "source/extensions/access_loggers/filters/cel/cel.h"
+
+#if defined(USE_CEL_PARSER)
+#include "parser/parser.h"
+#endif
+
+namespace Envoy {
+namespace Extensions {
+namespace AccessLoggers {
+namespace Filters {
+namespace CEL {
+
+Envoy::AccessLog::FilterPtr CELAccessLogExtensionFilterFactory::createFilter(
+    const envoy::config::accesslog::v3::ExtensionFilter& config, Runtime::Loader&,
+    Random::RandomGenerator&) {
+
+  // TODO: use factory_context validation. likely needs update to createFilter
```

Comment:
Missed this before: Our todo format is `// TODO(github_handle):` so to make it easier to track down the person with context.

Reply:
updated.